share/src/main/java/org/alfresco/web/site/servlet/config/AIMSConfig.java

```diff
@@ -44,18 +45,51 @@ public void init()
     public boolean isAIMSEnabled()
     {
         Config AIMSConfigCondition = this.configService.getConfig(AIMSConfigElement.AIMS_CONFIG_CONDITION);
```

Comment:
The name of the variable **AIMSConfigCondition** is a bit confusing. It can be easily confused with the name of a class. As per Java naming convention I think it should start with a lowercase letter. Maybe **aimsConfigCondition**?